app/models/searchgov_domain.rb

```diff
@@ -43,15 +42,12 @@ def index_sitemaps
   end
 
   def available?
-    /^200\b/ === (status || check_status)
+    !!((status || check_status) =~ /^200\b/)
```

Comment:
@jamesmadhoc , this can be improved with the new-ish `.match?` method:
```ruby
(status || check_status).match?(/^200\b/)
```
However, we can simplify it even further as:
```ruby
(status || check_status) == OK_STATUS
```
The pattern matching is a remnant of the early days of Garfield, when I wasn't sure if there would ever be variants of the `200 OK` string. 5 years worth of data later, I'm not concerned about that:
```ruby
> pp SearchgovDomain.pluck(:status).uniq.sort
["200 OK",
 "403 Forbidden",
 "404 Not Found",
 "SSL_connect returned=1 errno=0 state=error: certificate verify failed (Hostname mismatch)",
 "SSL_connect returned=1 errno=0 state=error: certificate verify failed (unable to get local issuer certificate)",
 "execution expired",
 "failed to connect: Connection refused - connect(2) for \"clu-in.org\" port 443",
 "failed to connect: Network is unreachable - connect(2) for \"clintonwhitehouse1.archives.gov\" port 443",
 "failed to connect: Network is unreachable - connect(2) for \"clintonwhitehouse2.archives.gov\" port 443",
 "failed to connect: getaddrinfo: Name or service not known"]
 ```